src/Microsoft.Crank.AzureDevOpsWorker/DevopsMessage.cs

```diff
@@ -34,10 +36,10 @@ public enum ResultTypes
         public string TaskInstanceId { get; set; }
         public string AuthToken { get; set; }
 
-        public Records Records { get; set; }
-        private DateTime _lastRecordsRefresh = DateTime.UtcNow;
         private static readonly JsonSerializerOptions _serializationOptions = new JsonSerializerOptions { PropertyNameCaseInsensitive = true };
 
+        private MemoryCache _memoryCache = new MemoryCache(new MemoryCacheOptions());
```

Comment:
Cache API result across Service Bus messages (benchmarks) since these are for the same pipeline.